src/main/java/com/browserstack/automate/ci/jenkins/local/JenkinsBrowserStackLocal.java

```diff
@@ -80,10 +82,33 @@ public JenkinsBrowserStackLocal(String accesskey, LocalConfig localConfig, Strin
     }
 
     public void start() throws Exception {
+        PluginLogger.log(logger,"Checking local options");
         Map<String, String> localOptions = new HashMap<String, String>();
         localOptions.put("key", accesskey);
-        if (binarypath != null && binarypath.length() > 0) localOptions.put("binarypath", binarypath);
-        super.start(localOptions);
+        
+        if(accesskey == null || accesskey.length()==0) {
+          PluginLogger.log(logger,"Access key is empty");
+        }else {
+          PluginLogger.log(logger,"Access key is not empty");
+        }
+        
+        if (binarypath != null && binarypath.length() > 0) {
```

Comment:
May be check for existence?

Reply:
The objective of the PR is to add logging. The code check for existence should be part of the existing local binary code.